mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
     """
     def __init__(self, regions=["ORD"]):
         """
         Create an API with the specified regions.
         """
         self._regions = regions
+        self.session_store = None
```

Comment:
`IAPIMock` providers should not store a reference to a session store.  Luckily it looks like if this was needed, the code that needed it has been eliminated, so it can be removed without causing problems.